docs/root/intro/arch_overview/http_routing.rst

```diff
@@ -87,6 +88,21 @@ headers <config_http_filters_router_headers_consumed>`. The following configurat
 Note that retries may be disabled depending on the contents of the :ref:`x-envoy-overloaded
 <config_http_filters_router_x-envoy-overloaded_consumed>`.
 
+.. _arch_overview_http_routing_hedging:
+
+Request Hedging
+---------------
+
+Envoy supports request hedging via specifying a :ref:`hedge policy
+<envoy_api_msg_route.HedgePolicy>`. This means that Envoy will race multiple
+simultaneous upstream requests and return the first valid response to the
+downstream according to retry policy.
```

Comment:
Reading this I couldn't tell if we wait for full response, or first successful-enough response headers.  Looks from code like we do the latter.  Could we clarify the docs on this point?